pkg/volume/gcepd/gce_pd.go

```diff
@@ -414,7 +414,9 @@ func (b *gcePersistentDiskMounter) SetUpAt(dir string, fsGroup *int64) error {
 	}
 
```

Comment:
Can you also add a successful mount message here?  It would be nice from the logs to be able to tell how much time the bind mount takes vs how much time the fsgroup setup takes.